Firmware/ControladorRobo/ControladorRobo.ino

```diff
@@ -81,23 +80,23 @@ ISR(TIMER1_COMPA_vect)
 {
     // Calculo das velocidades dos motores
 
-    float arcRight = ((float(pulsosDir))/3200.0)*M_PI;
-    float arcLeft = ((float(pulsosEsq))/3200.0)*M_PI;
+    float rightAngularDelta = ((float)right.read()/ENCODER_RESOLUTION)*M_PI;
+    float leftAngularDelta = ((float)left.read()/ENCODER_RESOLUTION)*M_PI;
 
-    wEsq = arcLeft*freq;
-    wDir = arcRight*freq;
+    float rightAngularSpeed = rightAngularDelta*freq;
+    float leftAngularSpeed = leftAngularDelta*freq;
 
-    posDir+=arcRight;
-    posEsq+=arcLeft;
+    rightAngPos += rightAngularDelta;
+    leftAngPos += leftAngularDelta;
 
-    pulsosDir = 0;
-    pulsosEsq = 0;
+    left.write(0);
+    right.write(0);
 
-    float u_esq = controle_esq.update(refEsq, wEsq);
-    float u_dir = controle_dir.update(refDir, wDir);
+    float u_esq = controle_esq.update(refEsq, leftAngularSpeed);
+    float u_dir = controle_dir.update(refDir, rightAngularSpeed);
 
     HBridge* bridge = &HBridge::self();
-    bridge->setWheelPWM(bridge->LEFT,(int)floor(u_dir*51));
-    bridge->setWheelPWM(bridge->RIGHT,(int)floor(u_esq*51));
+    bridge->setWheelPWM(bridge->LEFT,(int)floor(u_esq*51));
+    bridge->setWheelPWM(bridge->RIGHT,(int)floor(u_dir*51));
```

Comment:
This turns the 0-5 controller output into 0-255 number for the H-bridge.
But it's a "xunxo" and needs to be done properly